src/seedsigner/views/seed_views.py

```diff
@@ -2307,3 +2316,440 @@ def run(self):
+        
+        selected_seed = self.seeds[selected_menu_num]
+        
+        return Destination(RebuildSeedXORShowFingerprintView, view_args={"seed": selected_seed})
+
+
+
+class RebuildSeedXORViewShardsView(View):
+    """View all loaded shards in the rebuild seedXOR flow."""
+    
+    def __init__(self):
+        super().__init__()
+        self.shards = self.controller.storage.rebuild_seedxor_shards
+        
+    def run(self):
+        button_data = []
+        for i, shard in enumerate(self.shards):
+            network = self.settings.get_value(SettingsConstants.SETTING__NETWORK)
+            fingerprint = shard.get_fingerprint(network=network)
+            shard_num = i + 1
+            # TRANSLATOR_NOTE: Will insert the shard num and its fingerprint (e.g. "Shard #3: abcd1234")
+            button_data.append(ButtonOption("Shard #{}: {}".format(shard_num, fingerprint)))
+            
+        selected_menu_num = self.run_screen(
+            ButtonListScreen,
+            title=_("Loaded Shards"),
+            is_button_text_centered=False,
+            button_data=button_data
+        )
+        
+        return Destination(RebuildSeedXORManageView)
+
+
+
+class RebuildSeedXORRemoveShardsView(View):
+    """Remove shards from the rebuild seedXOR flow."""
+    
+    def __init__(self):
+        super().__init__()
+        self.shards = self.controller.storage.rebuild_seedxor_shards
+        
+    def run(self):
+        button_data = []
+        for i, shard in enumerate(self.shards):
+            network = self.settings.get_value(SettingsConstants.SETTING__NETWORK)
+            fingerprint = shard.get_fingerprint(network=network)
+            shard_num = i + 1
+            # TRANSLATOR_NOTE: Will insert the shard num and its fingerprint (e.g. "Shard #3: abcd1234")
+            button_data.append(ButtonOption("Shard #{}: {}".format(shard_num, fingerprint)))
+        
+        from seedsigner.gui.screens.screen import GUIConstants
+        selected_menu_num = self.run_screen(
+            ButtonListScreen,
+            title=_("Remove Shard"),
+            is_button_text_centered=False,
+            button_data=button_data,
+            button_selected_color=GUIConstants.ERROR_COLOR
+        )
+        
+        if selected_menu_num == RET_CODE__BACK_BUTTON:
+            return Destination(RebuildSeedXORManageView)
+        
+        self.controller.remove_rebuild_seedxor_shard(selected_menu_num)
+        return Destination(RebuildSeedXORManageView)
+
+
+
+class RebuildSeedXORCancelView(View):
+    """Confirm cancellation of rebuild seedXOR flow."""
+    
+    CONFIRM = ButtonOption("Confirm Cancel", button_label_color="red")
+    CONTINUE = ButtonOption("Continue SeedXOR")
+    
+    def run(self):
+        button_data = [self.CONTINUE, self.CONFIRM]
+        
+        num_shards = len(self.controller.storage.rebuild_seedxor_shards)
+        
+        text = _(
+            "Clear all loaded shards and return to Menu?"
```

Comment:
Should use the screen's title: "Home", not "Menu".

Should also have a `TRANSLATOR_NOTE` so that translators are aware that this is part of the Seed XOR flow so they'll understand what a "shard" is.